docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -365,7 +359,7 @@ To initialize the player for use:
      }
      ```
 
-     The playback is interrupted and the callback triggered when, for example, another multimedia application is run. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` according to the session policy. To get the current state, use the `player_get_state()` function.
+     The playback is interrupted and the callback is triggered when, for example, some other multimedia application starts. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` based on the resource management policy. To get the current state, use the `player_get_state()` function.
```

Comment:
I think the last part of [the @surbhig12's suggestion](https://github.com/Samsung/tizen-docs/pull/1059#discussion_r398315092) is missing.
Would you please check?

```suggestion
     The playback is interrupted and the callback is triggered when, for example, some other multimedia application starts. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` based on the resource management policy. To get the current state, use `player_get_state()`.
```

Reply:
I pushed new commit. thank you.